### PR TITLE
[0908] Amended apply filters button to top of filters panel

### PR DESCRIPTION
**Pull request by franciscobenedict-dfe**
### Context
apply filters button

### Changes proposed in this pull request

To improve usability, move the 'Apply filters' button to the top of the filters panel.

### Guidance to review

Same as https://github.com/DFE-Digital/publish-teacher-training/pull/3214
### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
Ported version of https://github.com/DFE-Digital/find-teacher-training/pull/1663 as the other was a fork and was causing issues

**Comment by defong** [comment]
[Move apply filters button to top of filters panel](https://trello.com/c/KVOpEbvF/908-move-apply-filters-button-to-top-of-filters-panel)

**Review by goodviber — APPROVED** [review]

**Review by victoriawinlow — APPROVED** [review]
Looks good to me

File: app/views/result_filters/_salary_filter.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<fieldset class="govuk-fieldset app-filter__group" data-qa="filters__funding">
+<fieldset class="govuk-fieldset app-filter__group no-hr govuk-!-margin-bottom-1" data-qa="filters__funding">
```
Comment (simonwhatley) [inline app/views/result_filters/_salary_filter.html.erb:1]: Why is `no-hr` needed?

**Review by simonwhatley — DISMISSED** [review]
I don't think there is a need for the `no-hr` style.

**Pull request merged**